README.md

```diff
@@ -107,6 +107,7 @@ To run tests, run the following command:
 ### Deployment
 
 You can deploy this project using:
+https://azeezumoru.github.io/Abdul-portfolio/
 
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224512497-727629e8-93f9-4cf4-afeb-c207e2ec94c3.png)

**[OPTIONAL]**

- [ ] It would be nice if you remove your Live Demo section since you don't have any live link to make your readme look professional and clean 😊 Once you will have a live link you can add the Getting Started Section with the live link in your readme file 👍 I noticed that you have your deployment link in the `Getting Started => Deployment` You can put that link here in this section 😊